muted-tests.yml

```diff
@@ -387,6 +387,9 @@ tests:
 - class: org.elasticsearch.xpack.ilm.actions.SearchableSnapshotActionIT
   method: testSearchableSnapshotsInHotPhasePinnedToHotNodes
   issue: https://github.com/elastic/elasticsearch/issues/125683
+- class: org.elasticsearch.xpack.test.rest.XPackRestIT
+  method: test {p0=deprecation/10_basic/Test Deprecations}
+  issue: https://github.com/elastic/elasticsearch/issues/123147
```

Comment:
I actually just opened #125682 to address the failing tests from that issue. The failure issue only references V7 compat tests, which will be fixed by my PR. We need a different fix for `main`. The fix is to specify `index: "*,-.security*"` here:
https://github.com/elastic/elasticsearch/blob/540f90e837e2fe30b25ebd4929bff225cceb42e6/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/deprecation/10_basic.yml#L18-L18

Could you make that change instead of muting it in this PR, please? Sorry for not mentioning that earlier, I was trying to do too many things at the same time...

Reply:
Done, let's see if it passes.

Reply:
Also thanks for fixing. I am wondering though if this is a fix or a workaround. What do you think?

Reply:
Talked offline and checked the proxy logs to see how it's used. We decided to anticipate the warning because the API is mainly used in this way `GET /_migration/deprecations`.